src/pcre2_jit_char_inc.h

```diff
@@ -1966,9 +1966,9 @@ switch(type)
     detect_partial_match(common, backtracks);
 
   if (type == OP_NOT_HSPACE)
-    read_char(common, 0x9, 0x3000, backtracks, READ_CHAR_UPDATE_STR_PTR);
+    read_char(common, 0x1, 0x3000, backtracks, READ_CHAR_UPDATE_STR_PTR);
```

Comment:
Maybe this could be changed to 0x0, since one byte utf8 characters may match.

Reply:
I wasn't sure why there was a min/max at all. I change it to `min=0x1, max=0x3000` to match some other calls in the file. What I do know is that the min/max values here need to chosen so that the HSPACE and VSPACE characters are included in the range, and in EBCDIC, `\t` is 0x5, which was causing test failures with the old code.